tests/api/core/strapi/document-service/validation/validation.test.api.ts

```diff
@@ -0,0 +1,242 @@
+import type { Core, Modules } from '@strapi/types';
+import { errors } from '@strapi/utils';
+import { createTestSetup, destroyTestSetup } from '../../../../utils/builder-helper';
+import resources from '../resources/index';
+import { ARTICLE_UID } from '../utils';
+
+let strapi: Core.Strapi;
+
+describe('Document Service Validations', () => {
+  let testUtils;
+
+  beforeAll(async () => {
+    testUtils = await createTestSetup(resources);
+    strapi = testUtils.strapi;
+  });
+
+  afterAll(async () => {
+    await destroyTestSetup(testUtils);
+  });
+
+  const methods = [
+    'findMany',
+    'findFirst',
+    'findOne',
+    'publish',
+    'delete',
+    'create',
+    'unpublish',
+    'clone',
+    'update',
+  ];
```

Comment:
should we add discard here?

Reply:
oops, I missed that one! Yep, I'll add it

Reply:
also, we support sending those params to `.count()` , so maybe we could test that one too? 🤔

Reply:
Done!